### PR TITLE
Get rid of default export in the data/utils module

### PR DESCRIPTION
**Pull request by jacksonrr3**

### CODE REVIEW

File: js/bundles/modules/data.js [hunk 1]
```diff
@@ -46,16 +46,14 @@ DevExpress.EndpointSelector = require('../../data/endpoint_selector');
 DevExpress.data.queryImpl = require('../../data/query_implementation').queryImpl;
 DevExpress.data.queryAdapters = require('../../data/query_adapters');
 
-const dataUtils = require('../../data/utils');
```
Comment (alexserov) [inline js/bundles/modules/data.js]: Are these changes necessary? Seems that we can keep an old version of this file

File: js/data/odata/query_adapter.js [hunk 2]
```diff
@@ -3,9 +3,20 @@ import { each } from '../../core/utils/iterator';
 import config from '../../core/config';
 import { extend } from '../../core/utils/extend';
 import queryAdapters from '../query_adapters';
-import { sendRequest, generateSelect, generateExpand, serializeValue, convertPrimitiveValue, serializePropName } from './utils';
```
Comment (alexserov) [inline js/data/odata/query_adapter.js:6]: @nightskylark We prefer single-line imports or multi-line ones?

File: js/bundles/modules/data.js (hunk 1 above)
Comment (jacksonrr3) [reply js/bundles/modules/data.js]: Not necessary, but explicit named import looks better.

**Review by alexserov — APPROVED** [review]
